src/game/game.h

```diff
@@ -552,17 +552,18 @@ struct clientstate
     int actortype, spawnpoint, ownernum, skill, points, frags, deaths, totalpoints, totalfrags, totaldeaths, spree, lasttimeplayed, timeplayed, cpmillis, cptime, queuepos;
     bool quarantine;
     string vanity;
-    vector<int> loadweap, lastweap, randweap;
+    std::vector<int> loadweap, lastweap, randweap;
     verinfo version;
 
     clientstate() : colour(0), model(0), checkpointspawn(1), weapselect(W_CLAW), lastdeath(0), lastspawn(0), lastpain(0), lastregen(0), lastregenamt(0), lastbuff(0), lastshoot(0),
         actortype(A_PLAYER), spawnpoint(-1), ownernum(-1), skill(0), points(0), frags(0), deaths(0), totalpoints(0), totalfrags(0), totaldeaths(0), spree(0), lasttimeplayed(0), timeplayed(0),
         cpmillis(0), cptime(0), queuepos(-1), quarantine(false)
     {
         vanity[0] = '\0';
-        loadweap.shrink(0);
-        lastweap.shrink(0);
-        randweap.shrink(0);
+        loadweap.clear();
```

Comment:
There is no reason for these calls in the constructor. All of them will be initialized empty, since `std::vector` has a default constructor.

Reply:
True that, solving